huey_multitenant/core.py

```diff
@@ -101,44 +101,51 @@ def load_config(self, conf_path):
             self._logger.error('Applications not configured in %s', conf_path)
             sys.exit(1)
 
-        all_conf = os.listdir(conf_path)
-        for conf in all_conf:
-            if conf.endswith('.conf'):
-                self._logger.info(conf)
-                try:
-                    parser = ConfigParser(defaults={
-                        'workers': '1',
-                        'worker-type': 'thread',
-                        'settings': None,
-                        'redis_host': 'localhost',
-                        'redis_port': '6379',
-                        'redis_prefix': None,
-                        'use_python3': 'false'
-                    })
-
-                    parser.read(os.path.join(conf_path, conf))
-                    for section in parser.sections():
-
-                        instance = HueyApplication(
-                            name=parser.get(section, 'redis_prefix'),
-                            python_path=parser.get(section, 'python'),
-                            script_path=parser.get(section, 'script'),
-                            workers=parser.get(section, 'workers'),
-                            worker_type=parser.get(section, 'worker-type'),
-                            settings=parser.get(section, 'settings'),
-                            redis_host=parser.get(section, 'redis_host'),
-                            redis_port=parser.get(section, 'redis_port'),
-                            redis_prefix=parser.get(section, 'redis_prefix') or section,
-                            use_python3=parser.getboolean(section, 'use_python3', fallback=False)
-                        )
-                        self.instances.append(instance)
-                except Exception as e:
-                    self._logger.exception('Error reading config %s', conf)
+        all_conf = (conf for conf in os.listdir(conf_path) if conf.endswith('.conf'))
 
+        pool = ThreadPoolExecutor(16)
+        instances = pool.map(lambda conf: self._load_instances_from_conf(conf, conf_path), all_conf)
+        for instance in instances:
+                if instance is None:
+                    continue
+                self.instances.append(instance)
```

Comment:
fijate que te quedo mal tabulado (por las dudas que te genere algun problema) y podes hacer 

self.instances = [instance for instance in instances if instance is not None]